src/api.c

```diff
@@ -38,6 +38,11 @@
 #include "helper.h"
 #include "system.h"
 
+/**
+ * @defgroup APIs
+ * @{
+ */
+
 #define API	__attribute__((visibility("default")))
```

Comment:
Is this explicitly pull in the `API` macro?  If so I think we can leave that out, the compiler/linker visibility attribute isn't something users/devs need to worry about, it's simply there to limit what symbols are visible in the shared library.

Reply:
Nope.

It seemed like a logical grouping to combine all of the API methods into a [Doxygen group](https://www.doxygen.nl/manual/commands.html#cmddefgroup).  Once I had them in a group, it provided an easy way to link to them in Doxygen.  Definitely open to other suggestions.

Reply:
Ah, okay, my mistake.

Somewhat related, is there a way to hit the `API` macro from the Doxygen docs?  It isn't really something a user/dev would need to worry about and I fear it could be misleading to some.

Reply:
The `API` macro is definitely accessible from Doxygen.  A user can view it either by going to Files -> Globals -> Macros or by searching for `API` directly.

https://drakenclimber.github.io/libseccomp/doxygen/wip/doxygen/globals_defs_a.html#index_a

https://drakenclimber.github.io/libseccomp/doxygen/wip/doxygen/group__APIs.html#gad8ce4efaa307683d3d763b37b4711c53

Thoughts?